src/community/libdeflate/src/main/java/org/geoserver/libdeflate/LibdeflateSettingsXStreamInitializer.java

```diff
@@ -0,0 +1,20 @@
+/* (c) 2020 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.libdeflate;
+
+import com.thoughtworks.xstream.XStream;
+import org.geoserver.config.util.XStreamPersister;
+import org.geoserver.config.util.XStreamPersisterInitializer;
+
+/** Persister initializer registering CogSettings related classes */
```

Comment:
Not CogSettings?

Reply:
My fault from copy and paste from the COG community module. Fixing it. Thanks for reporting.